script/stylelint-remove-disables.js

```diff
@@ -12,7 +12,7 @@ const REMOVED = `===REMOVED@${Date.now()}===`
 
 execFile('stylelint', ['--rd', '--quiet', '--formatter', 'json', process.argv[2]], (error, stdout) => {
 
-  // Filter to only needless diables comments
+  // Filter to only needless disables comments
```

Comment:
I'm really not sure what this is trying to say. The change here does agree with the filter call below, but I really don't understand the text here.